cmake/modules/FindMaya_Qt.cmake

```diff
@@ -0,0 +1,190 @@
+# Module to find Qt in Maya devkit.
```

Comment:
I documented this module, so I won't add extra comments here. Most of this came from what Dhruv wrote. I just made it generic and working on all platforms.